lacework/resource_lacework_alert_channel_splunk.go

```diff
@@ -0,0 +1,267 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkAlertChannelSplunk() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkAlertChannelSplunkCreate,
+		Read:   resourceLaceworkAlertChannelSplunkRead,
+		Update: resourceLaceworkAlertChannelSplunkUpdate,
+		Delete: resourceLaceworkAlertChannelSplunkDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkIntegration,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"intg_guid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"channel": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"hec_token": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"host": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"port": {
+				Type:     schema.TypeInt,
+				Required: true,
+			},
+			"ssl": {
+				Type:     schema.TypeBool,
+				Optional: true,
```

Comment:
If SSL is optional, could we be explicit on the default:
```suggestion
				Optional: true,
				Default:  false,
```